### ISSUE TITLE
Parsed Match is missing who got killed for first blood event

### ISSUE THREAD

**Issue by gu3st** [odota/core#1488]
Related to odota/ui#1042,  This match didn't seem to have a "killed" hero for the first blood event.  The first blood should have been the Drow (player_slot 2 IIRC), but nothing was returned.

Replay linked for analysis.

https://github.com/gu3st/replays/blob/master/3280538799.dem

**Comment by valeriilos** [comment]
True, stumbled upon the same issue. I did some more research though and what I found out is that some requests return "good" match data, which includes firstblood_claimed key and many more. And some requests return "broken" match data containing only a part of keys. 


[half-empty.txt](https://github.com/odota/core/files/1120356/half-empty.txt)
[half-empty2.txt](https://github.com/odota/core/files/1120358/half-empty2.txt)
[ok.txt](https://github.com/odota/core/files/1120359/ok.txt)
[ok2.txt](https://github.com/odota/core/files/1120357/ok2.txt)

**Comment by howardchung** [comment]
You're looking at unparsed matches (haven't had their replays downloaded and parsed yet).

**Comment by howardchung** [comment]
was this actually fixed?

**Comment by gu3st** [comment]
I was just cleaning up my github of stale issues that I'd opened.

**Comment by howardchung** [comment]
If it's actually still a problem, I think we should keep it open
